spec/workflow/context_spec.rb

```diff
@@ -19,7 +19,7 @@
     end
 
     context "with a simple string input" do
-      let(:input) { "\"foo\"" }
+      let(:input) { "foo" }
```

Comment:
For example. I think this is actually invalid input, right?

Reply:
it is invalid input for CLI.

But for the context, that is already inside the ruby world, so that should take ruby Strings.